app/javascript/lib/honeycrisp.js

```diff
@@ -103,16 +103,26 @@ var followUpQuestion = (function() {
                     }
                 });
 
+                let reset_other_follow_ups = $(this).attr('data-reset-other-follow-ups') !== "false";
```

Comment:
let's chat about this in the morning. my intention was that these attributes should be optional, and that the default should be the previous behavior, so that reset_other_follow_ups should be true unless the `data-reset-other-follow-ups` attribute is both present and exactly the value "false", but i'm not certain that i got it right

Reply:
oh I think i get it now! So most other use cases of followup questions never even have `data-reset-other-follow-ups` or `data-reset-hide-other-follow-ups` attributes, so they would always evaluate to true for `reset_other_follow_ups` & `hide_other_follow_ups` which retains previous behavior

For this particular feature, you wanted to _not_ hide/reset the followup questions on the page on a click of a question with a followup b/c you have multiple on a page that should remain open/shown and not reset. 

Took me a while to understand it but seems to make sense to me now!